Sources/Datadog/FeaturesIntegration/WebView/WebEventBridge.swift

```diff
@@ -47,9 +47,9 @@ internal class WebEventBridge {
         }
 
         if eventType == Constants.eventTypeLog {
-            logEventConsumer.consume(event: wrappedEvent, eventType: eventType)
+            try logEventConsumer?.consume(event: wrappedEvent, eventType: eventType)
         } else {
-            rumEventConsumer.consume(event: wrappedEvent, eventType: eventType)
+            try rumEventConsumer?.consume(event: wrappedEvent, eventType: eventType)
         }
```

Comment:
Because we pass Browser events to Logging / RUM feature storage, in case of Logging or RUM being disabled we will be sending Browser SDK events to void. This will result with loosing observability and entire set of telemetry from webviews, depending on native SDK configuration. This requires additional handling - @mariusc83 @buranmert @xgouchet what's our strategy for this?

I see 3 options:
* we can send events to their own storage - independent from Logging / RUM feature;
* we can inform the user on dropping event due to misconfiguration:
```swift
if eventType == Constants.eventTypeLog {
   if let logEventConsumer = logEventConsumer {
      try logEventConsumer.consume(event: wrappedEvent, eventType: eventType)
   } else {
      userLogger.warn(/* warn that event will be lost because Logging is disabled in native SDK */)
   }
} else {
   if let rumEventConsumer = rumEventConsumer {
      try rumEventConsumer(event: wrappedEvent, eventType: eventType)
   } else {
      userLogger.warn(/* warn that event will be lost because RUM is disabled in native SDK */)
   }
}
```
* we can register selectively for receiving only Logs / only RUM / both Logs and RUM events from Browser SDK.

Reply:
ℹ️ we decided to go with `userLogger.warn` option at today's daily

Reply:
@ncreated should we warn the user every time a new event is received (that might spam their console)?
or only once when `WebEventConsumer` is created?